remote/src/main/java/com/spotify/metrics/remote/OkRemote.java

```diff
@@ -89,4 +91,17 @@ public void onResponse(Call call, Response response) throws IOException {
         });
         return result;
     }
+
+    @Override
+    public void waitForAllCalls() {
+        Dispatcher dispatcher = client.dispatcher();
```

Comment:
Strangely the OkHttpClient doesn't seem to have any close method flushing all requests.